docs/run_test/azure_template.rst

```diff
@@ -0,0 +1,18 @@
+Azure Deployment Template
```

Comment:
It should be in the `write_test` parts. Add a link to reach this page in the `extension.rst`.

Reply:
I don't understand why it is put in extension.rst

Reply:
I think it's best place, but you can find another place.

Reply:
updated